pallets/handles/src/lib.rs

```diff
@@ -384,11 +419,12 @@ pub mod pallet {
 		/// * `Option<HandleResponse>` - The handle response if the MSA ID is valid.
 		///
 		pub fn get_handle_for_msa(msa_id: MessageSourceId) -> Option<HandleResponse> {
-			let full_handle = MSAIdToDisplayName::<T>::get(msa_id);
-			if full_handle.is_empty() {
+			let full_handle_option = MSAIdToDisplayName::<T>::get(msa_id);
+			if full_handle_option.is_none() {
 				return None
 			}
-
+			let full_handle_option = full_handle_option.unwrap();
```

Comment:
I think the handle fetching and extraction can just collapse to this? I guess it is the same number of lines in the end with the formatting.
```suggestion
			let full_handle = match MSAIdToDisplayName::<T>::get(msa_id) {
				Some((handle, _)) => handle,
				_ => {
					return None
				},
			};
```